.github/workflows/check.yaml

```diff
@@ -155,41 +152,27 @@ jobs:
         with: 
           toolchain: ${{ env.RUST_TOOLCHAIN }}
       - uses: Swatinem/rust-cache@v2
+      # install tool to convert cargo's JSON test output to JUNIT format
+      - run: |
+          cargo install cargo2junit
       # Using nextest because it's faster than built-in test
       - uses: taiki-e/install-action@nextest
       - name: Run cargo nextest
         run: |
-          cargo nextest run --message-format libtest-json-plus ${{ matrix.feature-flags }} > testresults${{ matrix.feature-flags }}.json
+          cargo nextest run ${{ matrix.feature-flags }} --profile ci
+      - name: Run doctests
+        run: |
+          # we use tee to let cargo test print to the console
+          RUSTC_BOOTSTRAP=1 cargo test --doc ${{ matrix.feature-flags }} -- -Z unstable-options --format json --report-time | tee target/doctest-results.json
+          # write output to same directory as the one used by cargo nextest in order
+          # to flatten the directory hierarchy when uploading the test results
+          cat target/doctest-results.json | cargo2junit > target/nextest/ci/doctest-results.xml
 
       - name: Upload all-features test results artifact
         id: test_results
         if: matrix.feature-flags == '--all-features'
         uses: actions/upload-artifact@v4
         with:
           name: test-results
-          path: testresults--all-features.json
-
-  # [impl->req~up-language-ci-test~1]
```

Comment:
In this scenario, where did you move the req implementation marker?

Reply:
It is already on the `nextest` job which now also includes running the doctests ...

Reply:
Ah I see